### PR TITLE
Documentation update

### LINKED ISSUE #49: Documentation
Hey @torch2424 just letting you know that I forked the documentation and currently rewrite the content of the readme, create some boilerplate pages and write-up a detailled but easy getting-started tutorial! I will probably push a PR later this day!

### PR DESCRIPTION
**Pull request by AndreasWebdev**
- Copied the readme and rewrote it
- Turned the supported filetypes under FAQ to a table
- Rewrote the getting started guide
- Rewrote the usage page so it's not just an output of `-help`
- Removed the invalid json comments in the settings and transformed it to a table for easier reading.

This resolves #49

### CODE REVIEW

**Comment by AndreasWebdev** [comment]
Don't get confused why I reverted the last commit, I forgot to change the branch on my end!

File: src/generate/FAQ.mdx [hunk 1]
```diff
@@ -8,19 +8,14 @@ route: /projectmanagement/FAQ
 
 ## What are the Supported File Types
 
-### Audio
+| Type | Supported Filetypes |
```
Comment (torch2424) [inline src/generate/FAQ.mdx]: 🔥 👍 Thanks for this!

File: src/generate/config.mdx [hunk 2]
```diff
@@ -6,237 +6,199 @@ route: /projectmanagement/config
...
-}
-```
-
-## Common Text Object Settings
-
-Common keys that all text objects support and are required to display the text
-
+This page will go in detail with every field of a projects `config.json`. If you ever need to reset the default settings, please check out the [default config.json](https://github.com/torch2424/live-stream-radio/blob/master/src/generate/template/config.json)
+
+## Settings Fields
```
Comment (torch2424) [inline src/generate/config.mdx]: Much better, thank you! 😄

File: src/generate/config.mdx [hunk 3]
```diff
@@ -6,237 +6,199 @@ route: /projectmanagement/config
...
+
+| Field | Usage |
+| - | - |
+| stream_url | The URL of the stream. You can use $stream_key as a variable and overwrite it with the `--output` CLI flag. |
```
Comment (torch2424) [inline src/generate/config.mdx]: Can you make it a little more clear that the $stream_key is replaces by the stream_key field?

In retrospect, should have just made it one field, but most Stream providers separate the logic, so just trying to follow that. Maybe in a later update we can do that 👍

File: src/index.mdx [hunk 4]
```diff
@@ -1,8 +1,31 @@
 ---
 name: Introduction
 route: /
+order: 1
```
Comment (torch2424) [inline src/index.mdx]: Oh nice! Didn't know that! This is my first time trying out docz.

File: src/index.mdx [hunk 5]
```diff
@@ -1,8 +1,31 @@
...
 ---
 
 # Introduction
+`live-stream-radio` is a 24/7 live stream radiostation CLI built with Node.js and powered by FFmpeg. This means that you'll be able to start a livestream radio for music, a looping gif/video, music information and other overlay items. This project is highly customizable and also features a REST HTTP JSON API for frontend interfaces.
```
Comment (torch2424) [inline src/index.mdx]: Beautiful intro page, thanks!

File: src/generate/config.mdx (hunk 3 above)
Comment (torch2424) [reply src/generate/config.mdx]: This is the only real change I want, the rest are compliments haha!

**Comment by torch2424** [comment]
Also, on your fork, you can run `npm run docz:deploy`, and that should create a `gh-pages` branch on your fork.

Feel free to open a PR from your `gh-pages` to the project `gh-pages` after this gets pulled, and the docs can go live 😄

**Comment by torch2424** [comment]
Thanks for making the docs 10000x better 😄

**Comment by AndreasWebdev** [comment]
I've also tried to deploy the gh-pages, however it seems as this is not possible, at least not without me looking into it further. docz builds with no issue, however it seems as the gh-pages command causes some ENOENT when publishing.

**Comment by torch2424** [comment]
Actually no worries! Was just about to deploy this for you! 😄

**Pull request merged**